app/views/admin/responses/show.html.erb

```diff
@@ -1,12 +1,18 @@
 <% page_title "#{@response.friendly_name.capitalize} for: #{@edition.title}" %>
 <% page_class "response" %>
 
+
```

Comment:
Tiny nit pick that I don't think this line needs to be added

Reply:
Ah I see it's removed in the next commit, feel free to ignore

Reply:
Ah i squashed it into the wrong commit 🤦 good catch! Thanks